AddParentGroup/function.json

```
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["put"],
+      "route": "groups/addParent/{id}/{parentId}"
```

```suggestion
      "route": "groups/parent/{id}/{parentId}"
```
If it's a PUT method, it's understood that it's meant to update the patent